### PR TITLE
Add custom user agent to job requests

### LINKED ISSUE #450: Send Custom User Agent
### Expected behavior
- *fullstop* runs a few jobs (I know of the two `Fetch*Jobs`) that act as web crawlers
- *fullstop* should send a custom `user-agent` header to let web servers know that a crawler arrives

### Step needed
- adapt `HttpGetRootCall` to send `user-agent` value `fullstop-job (https://github.com/zalando-stups/fullstop)`

### Dependencies
- none

### PR DESCRIPTION
**Pull request by lukasniemeier-zalando**
Fixes #450.

### CODE REVIEW

File: fullstop-jobs/src/main/java/org/zalando/stups/fullstop/jobs/config/HttpClientConfig.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+
+@Configuration
+public class HttpClientConfig {
+
+    @Bean
+    @Scope("prototype")
+    @SuppressWarnings("deprecation")
+    public CloseableHttpClient build() {
```
Comment (harti2006) [inline fullstop-jobs/src/main/java/org/zalando/stups/fullstop/jobs/config/HttpClientConfig.java]: Nice refactoring.

**Comment by harti2006** [comment]
:+1:

**Comment by lukasniemeier-zalando** [comment]
:+1:

**Comment by lukasniemeier-zalando** [comment]
:+1:

**Comment by ChristianLohmann** [comment]
👍

**Pull request merged**